pkg/handler/handler.go

```diff
@@ -521,9 +538,11 @@ func (h handler) unbind(w http.ResponseWriter, r *http.Request, params map[strin
 			})
 			return
 		}
-		if ok, status, err := h.validateUser(userInfo.Username, serviceInstance.Context.Namespace); !ok {
-			writeResponse(w, status, broker.ErrorResponse{Description: err.Error()})
-			return
+		if !nsDeleted {
```

Comment:
Don't we need to check and not run the pod during unbind like deprovision?

Reply:
I haven't put this in for now, since I believe launch_apb_on_bind would have to be turned on.  I can add it, as long as we think it's worth doing now.

Reply:
Fixed